### ISSUE TITLE
Tool tips are not appearing for Explore, Search, Source Control, Run and Debug, Extensions, Account and Settings Tab controls as soon as Keyboard focus Lands on controls: A11y_VisualStudioCodeClient_ Menu bar Options_Keyboard.

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/vscode#250997]
Please do not close this bug. This bug should only be closed by Trusted Tester after verification.
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #BM_VisualStudioCodeClient_Win32_Jun2025; #Visual Studio Code Client; #WCAG2.1.1; #FTP; #Keyboard; #AILimited; #Win11; #Benchmark; #VSCode2025;

### Environment Details:
App Name: Visual Studio Code Client
#URL: Visual Studio Code Insider
Microsoft Windows 11 Enterprise
OS Build: 26100.4202

### Repro Steps:

1. Open Visual Studio Code Insider editor.
2. Start navigating with Tab key from all the Menu items.
3. Once the Keyboard focus lands on Explore tab item Observe that Tool tips are not appearing for Explore, Search, Source Control, Run and Debug, Extensions, Account and Settings Tab controls as soon as Keyboard focus Lands on controls.

### Actual Result:
Tool tips are not appearing for Explore, Search, Source Control, Run and Debug, Extensions, Account and Settings Tab controls as soon as Keyboard focus Lands on controls

### Expected Results:
Tool tips should be appearing for Explore, Search, Source Control, Run and Debug, Extensions, Account and Settings Tab controls as soon as Keyboard focus Lands on controls.

### User Impact:
Keyboard users wont be able to get the tooltip information if tooltips does not appear as soon as keyboard focus lands on it.

### Attachment:

https://github.com/user-attachments/assets/6ad682fd-26fe-4d8b-aff0-f8459cecc912

**Comment by benibenj** [comment]
@Tyriar this regression was caused by the adoption of the new `setupDelayedHover` API as it does not support triggering on focus

**Comment by benibenj** [comment]
We could possibly do something like this, however, it seems like there still is some issues when moving focus between the `Accounts` action and the last view pane action using the arrow keys.

```typescript
store.add(addDisposableListener(target, EventType.FOCUS, e => {
	this.showInstantHover(resolveHoverOptions(), false, true);
}, true));
store.add(addDisposableListener(target, EventType.BLUR, e => {
	this.hideHover();
}, true));
```

**Comment by Tyriar** [comment]
This works for me, see it's read by screen reader:

![Image](https://github.com/user-attachments/assets/91804eb0-6760-4e15-83e2-adbf4263c007)

ctrl+k, ctrl+i shows tooltip correctly:

![Image](https://github.com/user-attachments/assets/d165f411-0342-4832-94e3-db480eafa3f7)

AFAIK there's no requirement that we show the tooltip on focus?

**Comment by meganrogge** [comment]
@keerthiduvvuri where's the a11y guideline that mandates a label must be shown on keyboard focus of an element? Given there's a way to do this that's well known via `ctrl+k+i`, I think we can close this?

**Comment by keerthiduvvuri** [comment]
The shortcut key is documented https://code.visualstudio.com/shortcuts/[keyboard-shortcuts-windows.pdf](https://code.visualstudio.com/shortcuts/keyboard-shortcuts-windows.pdf), and also when user is using the Ctrl+K+Ctrl+I, tooltip is Visible. Hence closing the bug.
